### ISSUE TITLE
Define the visually equivalency.

### ISSUE THREAD

**Issue by rniwa** [w3c/selection-api#6]
We need to define the concept of visually equivalency used in the definition of Selection.containsNode.

**Comment by yosinch** [comment]
I would like to omit visually equivalency from containsNode. Selection.containsNode() should work on DOM tree and Flat tree.

CSSOM should have rectangle or quad based node containment predicate.
Visual based containment on Selection can write as:

// methods are hypothetical 
const nodeQuad = node.getClientQuad()
for (const range : selection.ranges) {
  const rangeQuad = range.getClientQuad();
  if (rangeQuad.intersects(nodeQuad))
   return true;
}
return false;

**Comment by rniwa** [comment]
I don't think we can make `containsNode` not rely on visual equivalency due to backwards compatibility requirements.
